src/common/transformations/src/transformations/common_optimizations/nop_elimination.cpp

```diff
@@ -555,6 +555,24 @@ pass::EliminateConcatStridedSlice::EliminateConcatStridedSlice() {
                     return false;
                 }
 
+                // check that concatenated and split axis is the same
+                auto check_axis = [concat_axis](const std::vector<int64_t>& masks) {
+                    for (size_t axis = 0; axis < masks.size(); ++axis) {
+                        if (masks[axis] != 1 && axis != (size_t)concat_axis) {
+                            return false;
+                        }
+                        if (masks[axis] != 0 && axis == (size_t)concat_axis) {
+                            return false;
+                        }
+                    }
+                    return true;
+                };
+                auto begin_mask = strided_slice_node->get_begin_mask();
+                auto end_mask = strided_slice_node->get_begin_mask();
```

Comment:
```suggestion
                auto end_mask = strided_slice_node->get_end_mask();
```

Reply:
good catch!

Reply:
@zhaixuejun1993 , could you please add a test which would reveal this typo? (Could be done at a later stage, after we merge it.)